utils/images.ahk

```diff
@@ -27,3 +27,9 @@ GetCurrentMapPage() {
         }
     }
 }
+
+SearchUpgrade(path) {
+	if (SearchImage("buttons\upgrades\" path "R", "", 250, (325+path*150), 1494, (349+path*150)) or SearchImage("buttons\upgrades\" path "L", "", 250, (325+path*150), 1494, (349+path*150))) {
```

Comment:
Indentation is a tab instead of four spaces and this is quite a long line that should probably be split to two.